scripts/uploadToLokalise.js

```diff
@@ -0,0 +1,35 @@
+const { LokaliseApi } = require('@lokalise/node-api');
+
+async function uploadToLokalise() {
+  try {
+    const lokaliseToken = process.env.TOKEN;
+    const branchName = process.env.BRANCH_NAME.replace('refs/heads/', '');
+    const projectId = process.env.PROJECT_ID;
+    const files = process.env.FILES.split(' ');
+    const lokalise = new LokaliseApi({ apiKey: lokaliseToken });
+
+    if (!files.length > 0) {
+      const promises = [];
+      files.forEach(path => {
```

Comment:
I believe you'll get all changed files here,  you have to only upload translation `-en` files